xbmc/ApplicationPlayer.h

```diff
@@ -29,6 +29,11 @@ struct VideoStreamInfo;
 struct SubtitleStreamInfo;
 struct TextCacheStruct_t;
 
+namespace PLAYLIST
+{
+enum class PlaylistId;
```

Comment:
This redefinition is unnecessary if included `playlists/PlayListTypes.h` in this file

Reply:
I agree, although in this case there is no cost to include it here, since it is very light and above all because it is also included in `ApplicationPlayer.cpp` anyway (so can be moved here).

In any case, I leave at @CastagnaIT choose.